v3-docs/docs/api/accounts.md

```diff
@@ -888,6 +888,8 @@ email with a link the user can use to verify their email address.
 
 <ApiBox name="Accounts.removeEmail" />
 
+<ApiBox name="Accounts.replaceEmail" />
```

Comment:
You implemented the async version directly, shouldn't it be named Accounts.replaceEmailAsync instead? The same applies elsewhere.